ConsultFlow/ConsultFlow.Core.Domain/Entities/IdentityProviders/IdentityProviderType.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace ConsultFlow.Core.Domain.Entities.IdentityProviders
+{
+    public enum IdentityProviderType
+    {
+        LOCAL = 1
```

Comment:
@fwcheesecake we should use PascalCase for enums, like you did in `LoginType` enum